### DISCUSSION TITLE
QuickSelect - hiding the hint line

### DISCUSSION THREAD

**Discussion by j-xella** [wezterm/wezterm#4632] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20230408-112425-69ae8472 x86_64-pc-windows-msvc

### Ask your question!

When quick select mode is enabled, a hint is shown on the bottom line, something like this:

`Select: type highlighted prefix to copy ....`

This hint is very helpful, but I know it by heart now. Also, it hides the bottom line from selection. This is important, for example when vim is full screen, the results of commands are displayed on the bottom line, and sometimes I would like to quick copy them.

Of course, I can just use normal selection for the bottom line, but QuickSelect is, well, quick, and it would be nice to be able to use it.

Is there a way to

* hide the QucikSelect hint
* Toggle it during the quick select mode
* If there is a selectable text on the bottom line, show it on top of the hint line

**Comment by wez** [comment]
no, sorry. This is tracked in: https://github.com/wez/wezterm/issues/2693
